Sasquatch/SasquatchSwift/AppDelegate.swift

```diff
@@ -30,7 +30,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
   private var locationManager : CLLocationManager = CLLocationManager()
 
   var window: UIWindow?
-
+  func topMostViewController() -> UIViewController? {
+    let window = UIApplication.shared.keyWindow
```

Comment:
this variable is not needed. You already have a defined ivar `window` on line 32